types/youtube/index.d.ts

```diff
@@ -518,6 +535,11 @@ declare namespace YT
 		 * Comma separated list of video IDs to play after the URL path's video.
 		 */
 		playlist?: string;
+		
+		/**
+		 * Whether videos play inline or fullscreen in an HTML5 player on iOS. (currently by default, Fullscreen).
+		 */
+		playsinline?: PlaysInline;
```

Comment:
To follow up, you can leave the const enum as a declaration, but it shouldn't be used here for the type of `playsinline`.